app/src/ui/open-pull-request/pull-request-diff.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
+import { IDiff, ImageDiffType } from '../../models/diff'
+import { Repository } from '../../models/repository'
+import { CommittedFileChange } from '../../models/status'
+import { SeamlessDiffSwitcher } from '../diff/seamless-diff-switcher'
+import { Dispatcher } from '../dispatcher'
+import { openFile } from '../lib/open-file'
+
+interface IPullRequestDiffProps {
+  readonly repository: Repository
+  readonly dispatcher: Dispatcher
```

Comment:
I am not a huge fan of passing the dispatcher down far into the component structure but there were a few dispatcher methods that would only pertain to diffing and felt muddy at the dialog level (which I am trying to reserve for the purpose of managing the tab state). Since we don't have a strict convention to not passing the dispatcher down, I think it is cleaner to go ahead and pass it down here.